cypress/integration/dashboard_spec.js

```diff
@@ -0,0 +1,52 @@
+describe('Testing environment', () => {
+  it('Should expect true to be true', () => {
+    expect(true).to.equal(true);
+  });
+});
+
+describe('Dashboard user flows', () => {
+  it('Should be able to visit http://localhost:3000 and see the header and list of movie posters', () => {
+
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+      movies: [
+        {
+          id: 694919,
+          poster_path: 'https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg',
+          backdrop_path: 'https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg',
+          title: 'Money Plane',
+          average_rating: 6.142857142857143,
+          release_date: '2020-09-29'
+        }, {
+          id: 337401,
+          poster_path: 'https://image.tmdb.org/t/p/original//aKx1ARwG55zZ0GpRvU2WrGrCG9o.jpg',
+          backdrop_path: 'https://image.tmdb.org/t/p/original//zzWGRw277MNoCs3zhyG3YmYQsXv.jpg',
+          title: 'Mulan',
+          average_rating: 5.2727272727272725,
+          release_date: '2020-09-04'
+        }, {
+          id: 718444,
+          poster_path: 'https://image.tmdb.org/t/p/original//uOw5JD8IlD546feZ6oxbIjvN66P.jpg',
+          backdrop_path: 'https://image.tmdb.org/t/p/original//x4UkhIQuHIJyeeOTdcbZ3t3gBSa.jpg',
+          title: 'Rogue',
+          average_rating: 7,
+          release_date: '2020-08-20'
+        }
+      ]
+    });
+
+    cy.visit('http://localhost:3000')
+      .get('h1').contains('Rancid Tomatillos')
+      .get('h2').contains('Featured Films')
+      .get('img[id="694919"]').should('be.visible')
```

Comment:
is bracket notation necessary here?

Reply:
Yup, this is cypress syntax targeting the image with the specified id